independent-publisher-connectors/Mensagia/apiProperties.json

```diff
@@ -0,0 +1,25 @@
+{
+    "properties": {
+        "connectionParameters": {
+            "api_key": {
+                "type": "securestring",
+                "uiDefinition": {
+                    "displayName": "Api Key",
+                    "description": "Api Key for this API.",
+                    "tooltip": "Set your Api Key.",
+                    "constraints": {
+                        "tabIndex": 2,
+                        "clearText": false,
+                        "required": "true"
+                    }
+                }
+            }
+        },
+        "iconBrandColor": "#f8f8f8",
+        "capabilities": [
+            "actions"
+        ],
+        "publisher": "Sistemes Informàtics ICON, S.L.",
```

Comment:
@AfriSR The publisher value has unicodes which may not be supported in all the Power Automate pages, Can you please replace all the unicodes with respective characters in the English language.